src/frontend/AppStack.tsx

```diff
@@ -24,6 +23,9 @@ import LanguageSettings from "./screens/Settings/LanguageSettings";
 import CoordinateFormat from "./screens/Settings/CoordinateFormat";
 import Experiments from "./screens/Settings/Experiments";
 import HomeHeader from "./sharedComponents/HomeHeader";
+import { LeaveProjectScreen } from "./screens/LeaveProject/LeaveProject";
+import { LeaveProjectProgress } from "./screens/LeaveProject/LeaveProjectProgess";
+import { LeaveProjectCompleted } from "./screens/LeaveProject/LeaveProjectCompleted";
```

Comment:
more of an aside but i'm all for named exports but we should stick to a convention (e.g. named exports or default exports for React components)

Reply:
Im going to resolve this for now, but yes this is something we should talk about!